src/dagster_ray/configs.py

```diff
@@ -40,11 +40,11 @@ class RayExecutionConfig(dg.Config):
     resources: dict[str, float] | None = Field(default=None, description="Custom resources to allocate.")
 
     @classmethod
-    def from_tags(cls, tags: Mapping[str, str]) -> RayExecutionConfig:
+    def from_tags(cls, address, tags: Mapping[str, str]) -> RayExecutionConfig:
```

Comment:
I believe this method is the wrong place to handle `address` as this class  - `RayExecutionConfig` - currently doesn't have such a field. 

If you'd want to change how `address` is handled, you'd have to modify `RayRunLauncherConfig` or `RayExecutorConfig` instead.